tuf/client_rework/updater_rework.py

```diff
@@ -35,27 +36,26 @@ class Updater:
     """
     Provides a class that can download target files securely.
 
-    Attributes:
-        metadata:
-
-        repository_name:
-
-        mirrors:
-
-        fetcher:
-
-        consistent_snapshot:
+    TODO
     """
 
     def __init__(
         self,
         repository_name: str,
-        repository_mirrors: Dict,
+        metadata_url: str,
+        default_target_url: Optional[str] = None,
```

Comment:
I know this makes the variable names _really_ long, but should we better indicate that these are base urls? Or will we rely on the docstring for that?